helm_deploy/values-preprod.yaml

```diff
@@ -0,0 +1,37 @@
+# This is a YAML-formatted file.
```

Comment:
I've used two approaches to IaC in the past - config in the repo and config separate from the repo. Both have advantages and drawbacks:

1. In repo
  - advantage - simpler to manage and easier to find and identify
  - disadvantage - ties infrastructure to application code, which means reverts of infrastructure will require roll-forwards, or reverting of features
2. Out of repo
  - advantage - separates infrastructure from code, allowing infra changes to occur without changing application artefact
  - disadvantage - requires more complicated orchestration.

Just to be clear, are we explicitly intending on moving from a flavour of type 2, to type 1? And, does match with how we "do devops" within our group and the wider HMPPS, and the even wider MOJ?

Reply:
There are no set ways of working- I think it's really what works best for your development team.  You have outlined the arguments nicely.  Moving to helm deployments doesn't exclude having option 1.  In a future iteration you could move the deployment config elsewhere.

Reply:
This is achieving two things (or maybe more):
1. Changing how we deploy so it's no longer a kubectl set/apply command, but uses helm. 
2. Moving it to this repo following the way it's done currently in DPS. 
It does tie the infra to the application code, and as Matt says, if that becomes a continuous problem, we can change it.